### ISSUE TITLE
Add language injections for Neo4j Java Driver

### ISSUE THREAD

**Issue by ojhughes** [albertoventurini/graphdb-intellij-plugin#29]
First off, big thanks for forking this and making so many improvements. I had also forked the project and was working through the painful process of fixing issues. I see you have got quite a bit further along than me so I will contribute my efforts to this repo instead.

In my fork I had started adding language injections for the Java driver eg .

```
    <injection language="Cypher" injector-id="java">
        <display-name>Neo4j - Bolt Java Driver (org.neo4j.driver)</display-name>
        <place>
            <![CDATA[
            psiParameter().ofMethod(0, psiMethod().withName("run")
                    .withParameters("java.lang.String")
                    .definedInClass("org.neo4j.driver.QueryRunner"))
            ]]>
        </place>
    </injection>
```

It would be nice to also add injections for other languages such as JavaScript and Go

**Comment by albertoventurini** [comment]
Hi @ojhughes, awesome, I am happy to join forces on this plugin!

If I understand this correctly, this would automatically inject Cypher support (i.e. syntax highlighting, autocompletion etc.) on strings that are passed as parameters to the `run()` method, right? That's definitely a nice feature to have.

**Comment by ojhughes** [comment]
Yes that's right. It currently works for some libraries like OGM but not the drivers
